conf/messages

```diff
@@ -1,2 +1,8 @@
-service.name = customs-service-status-frontend
-service.text = This is your new service
+service.name = Customs Service Status Frontend
```

Comment:
this should be `Customs service availability` if this name is for the header

Reply:
done